sky/utils/command_runner.py

```diff
@@ -448,13 +448,11 @@ def run(
             proc = subprocess_utils.run(command, shell=False, check=False)
             return proc.returncode, '', ''
 
-        command_str = self._get_command_to_run(
-            cmd,
-            process_stream,
-            separate_stderr,
-            # +1 to skip first new line.
-            skip_lines=skip_lines + 1,
-            source_bashrc=source_bashrc)
+        command_str = self._get_command_to_run(cmd,
+                                               process_stream,
+                                               separate_stderr,
+                                               skip_lines=skip_lines,
```

Comment:
should we change other usage of `skip_lines`?

Reply:
What do you mean by other usage of skip_lines?

Reply:
Oh nvm. I mean `skip_lines=5 if source_bashrc else 3` in our codebase, but just found out that is already fixed by #3586 .
<img width="735" alt="image" src="https://github.com/skypilot-org/skypilot/assets/74357442/2e5c6176-1524-4107-b958-8b5814ca1445">